sql/character.anticheat.sql

```diff
@@ -45,6 +45,18 @@ CREATE TABLE IF NOT EXISTS `players_reports_status` (
   PRIMARY KEY (`guid`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
 
+-- Dumping structure for table characters.lua_cheaters
+DROP TABLE IF EXISTS `lua_cheaters`;
+CREATE TABLE IF NOT EXISTS `lua_cheaters` (
+  `guid` int(11) NOT NULL,
```

Comment:
Please use the same type as other tables, so
```sql
`guid` int unsigned NOT NULL DEFAULT 0,
```

Reply:
Ah. Thats what i forgot. I couldnt remember the past month what i had left to do. Ill get to it later.